### PR TITLE
Disable autocomplete

### LINKED ISSUE #223: Remove autofill on form in pregnancies/edit
Flagged by @lwaldsc and @NerdyGirl537 :

* On `pregnancies/:id/edit`, autofill dropdown menu shows up:
  Thing I don't like: Chrome Autofill form options show up 
  Thing I would like better: Remove autofill form options

This is probably as simple as adding an HTML5 option to disable autofills. Should be fairly straightfoward!

### PR DESCRIPTION
**Pull request by CamLatimer**
I rule and have completed some work on Case Manager that's ready for review!

This pull request makes the following changes:
* disables form autocomplete feature 

It relates to the following issue #s: 
* Fixes #223

### CODE REVIEW

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -5,22 +5,22 @@
     <%= devise_error_messages! %>
 
     <h3>Change your account details</h3>
-    <%= f.email_field :email, autofocus: true %>
+    <%= f.email_field :email %>
```
Comment (colinxfleming) [inline app/views/devise/registrations/edit.html.erb]: hey @CamLatimer -- can you restore these extra attributes? We want to take autocomplete off the pregnancy forms, but we do want to make sure it stays on the account edit form.

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -3,14 +3,13 @@
 
   <%= bootstrap_form_for resource, as: resource_name, url: registration_path(resource_name) do |f| %>
     <%= devise_error_messages! %>
-    
+
     <%= f.email_field :email, autofocus: true %>
     <%= f.text_field :name %>
-    <%= f.password_field :password, autocomplete: "off", label: "Password (#{@minimum_password_length || 8} characters minimum)" %>
-    <%= f.password_field :password_confirmation, autocomplete: "off" %>
+    <%= f.password_field :password, label: "Password (#{@minimum_password_length || 8} characters minimum)" %>
```
Comment (colinxfleming) [inline app/views/devise/registrations/new.html.erb]: @CamLatimer same deal, can you revert these changes so the `autocomplete=off` comes back?

File: app/views/pregnancies/_abortion_information.html.erb [hunk 3]
```diff
@@ -3,27 +3,27 @@
     <h2>Abortion information</h2>
 
     <div class="info-form-left col-md-6">
-      <%= f.fields_for (pregnancy.clinic || pregnancy.build_clinic) do |cl| %> <!-- was pregnancy.clinic, but that eats muds if clinic is nil  --> 
+      <%= f.fields_for (pregnancy.clinic || pregnancy.build_clinic) do |cl| %> <!-- was pregnancy.clinic, but that eats muds if clinic is nil  -->
         <h4>Clinic details</h4>
-        <%= cl.text_field :name, label: 'Clinic name:' %>
-        <%= cl.text_field :street_address_1, label: 'Street address 1:' %>
-        <%= cl.text_field :street_address_2, label: 'Street address 2:' %>
+        <%= cl.text_field :name, label: 'Clinic name:', autocomplete: 'off' %>
```
Comment (colinxfleming) [inline app/views/pregnancies/_abortion_information.html.erb]: 💯

File: app/views/pregnancies/_patient_dashboard.html.erb [hunk 4]
```diff
@@ -1,16 +1,16 @@
-<div class="col-md-4">
+<div class="col-md-4" >
```
Comment (colinxfleming) [inline app/views/pregnancies/_patient_dashboard.html.erb]: Lookin' good @CamLatimer !

**Comment by colinxfleming** [comment]
Hey @CamLatimer ! Had a few suggestions to implement before merge, let me know if you have any questions. 

Thanks much for handling this!

**Comment by CamLatimer** [comment]
@colinxfleming, I added back the autofocus attribute and left out the autocomplete attribute.  Those files are now back to how they were when I first pulled them down locally to my laptop. Here is the last pushed commit: 

https://github.com/CamLatimer/dcaf_case_management/commit/b64d43086ceaa181ab74a4676854ea9bc0641ad1

Let me know if that's okay.

**Comment by colinxfleming** [comment]
Thanks much @CamLatimer !
